_episodes/05-memory.md

```diff
@@ -211,36 +225,42 @@ separate information will need to be "stored" in memory during each part of the
 
 ## Why Guided Practice is Important
 
-The quality of practice that is provided to learners is very important. In Carpentry
-workshops, we use *guided practice*: we set up a structure in which learners can test their
-skills and get feedback on their progress. This contrasts with another teaching strategy
-variously known as *constructivist*, *discovery*, *problem-based*, *experiential*
-or *inquiry-based* learning.
-
-In 2006, these types of teaching were the subject of a paper titled "[Why Minimal Guidance During Instruction Does Not Work: An Analysis of the Failure of Constructivist, Discovery, Problem-Based, Experiential, and Inquiry-Based Teaching][kirschner-paper]".
-
-This paper argued that minimal-guidance approaches to teaching ignore both what we know
-about human cognitive architecture and evidence from many research studies showing that
-minimally-guided instruction is less effective than guided learning.
-
-Minimally-guided instruction requires learners to simultaneously
-master a domain's factual content
-and its search and problem-solving strategies.
-Fostering creativity and independence is intuitively appealing,
-but that doesn't mean it works.
-
-One of the paper's authors (Sweller) proposed an alternative
-based on the theory of *[cognitive load][wikipedia-cognitive-load]*.
+Another way to think about the challenge of managing memory is through
+the theory of *[cognitive load][wikipedia-cognitive-load]*.
 It posits that people have to deal with three things when they're learning:
 
 *   *Intrinsic* load is what they have to keep in mind in order to carry out a learning task.
 *   *Germane* load is the (desirable) mental effort required to create linkages between new information and old
     (which is one of the things that distinguishes learning from memorization).
 *   *Extraneous* load is everything else that distracts or gets in the way.
 
-Cognitive load theory's proponents claim that eliminating extraneous cognitive load accelerates learning.
+Cognitive load theory's proponents claim that eliminating extraneous cognitive load accelerates learning. One
+way to do this is by using *guided practice*: we set up a structure in which learners can test their
+skills and get feedback on their progress.
+
+> ## Why We Don't Use Minimal Guidance
+>
+> An alternative to guided practice is a *minimal guidance* approach, where
+> learners are given raw materials (for example a text or reference) and
+> asked to explore and learn to solve problems on their own.
+> Minimal guidance practice comes from another teaching strategy
+> variously known as *constructivist*, *discovery*, *problem-based*, *experiential*
+> or *inquiry-based* learning.
+>
```

Comment:
Maybe add "You may have heard of some of these strategies, and they are not without merit! But they frequently run aground, and when they do it tends to be because of something we call *cognitive overload*.
Then... "Cognitive load theory, introduced by Kirshner et al. (link) in response to the popularity and uneven success of these approaches, applies what we know about human cognitive architecture to understand why minimal guidance often fails." in place of the paragraph below, leading up to line 255.

Reply:
I wasn't sure how much of the paragraph you wanted to change, let me know what you think when I push the update.

Reply:
I think I just accidentally committed an adjustment to this directly to your branch. GitHub is being weird because that definitely was not my selected option. Anyway, feel free to undo or put the paper title back in or whatever!